private/sass/sites/_custom.scss

```diff
@@ -30,28 +30,30 @@
 }
 .spacer:after,
 .spacer-xs:after,
+.spacer-sm:after,
+.spacer-md:after,
 .spacer-lg:after,
 .spacer-zero:after {
-    clear: both;
     content: "";
     display: table;
+    clear: both;
 }
 
-// #HELPERS/accessibility#
+// HELPERS/accessibility
 .skip-links {
     @extend .list-unstyled;
     position: absolute;
-    z-index: 9999;
     top: $navbar-height;
     right: 0;
     left: 0;
+    z-index: 9999;
```

Comment:
actually yes, this is copied from the xplorer theme, needs to be addressed there, ticket will be created